etc/WMAgentConfig.py

```diff
@@ -141,6 +141,7 @@
 # this has to be unique for different work queue. This is just place holder
 config.WorkQueueManager.queueParams["QueueURL"] = "http://%s:5984" % (config.Agent.hostName)
 config.WorkQueueManager.queueParams["WorkPerCycle"] = 100  # don't pull more than this number of elements per cycle
+config.WorkQueueManager.queueParams["drain_mode"] = False
```

Comment:
ah ha, I had that originally but changed it with this commit as I thought I only saw drain_mode. I'll change it back.